ddsketch/mapping.py

```diff
@@ -169,14 +175,12 @@ def _cubic_exp2_approx(self, value):
             - 9 * self.A * self.B * self.C
             - 27 * self.A * self.A * (value - exponent)
         )
-        cardano = np.cbrt(
-            (delta_1 - np.sqrt(delta_1 * delta_1 - 4 * delta_0 * delta_0 * delta_0)) / 2
-        )
+        cardano = _cbrt((delta_1 - math.sqrt(delta_1 * delta_1 - 4 * delta_0 * delta_0 * delta_0)) / 2)
```

Comment:
~~~
In [7]: %timeit 10 ** .5
7.2 ns ± 0.116 ns per loop (mean ± std. dev. of 7 runs, 100000000 loops each)

In [8]: %timeit math.sqrt(10)
68.6 ns ± 1.15 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

In [9]: assert 10 ** .5 == math.sqrt(10)
~~~